promql/bench_test.go

```diff
@@ -171,6 +171,12 @@ func BenchmarkRangeQuery(b *testing.B) {
 		{
 			expr: "histogram_quantile(0.9, rate(h_X[5m]))",
 		},
+		// Extremely large queries
```

Comment:
Full stop

And it's more sub queries with lots of steps, a 4s query isn't large.